src/PKSim.Core/Services/ParticleApplicationCreator.cs

```diff
@@ -77,6 +75,12 @@ private void addDrugStartFormulaForParticles(IApplicationBuilder applicationBuil
             setupParticleBin(applicationBuilder, i, binRadius[i - 1], binNumberOfParticlesFactor[i - 1], formulaCache);
          }
 
+         //// add insoluble molecule for particle bin
+         //var appMoleculeBuilder = _objectBaseFactory.Create<IApplicationMoleculeBuilder>().WithName(insolubleMoleculeName(0)); //TODO replace insolubleMoleculeName(0) with const string
+         //appMoleculeBuilder.RelativeContainerPath = _objectPathFactory.CreateObjectPathFrom(CoreConstants.ContainerName.InsolubleDrug);
+         //appMoleculeBuilder.Formula = insolubleDrugStartFormula(formulaCache);
+         //applicationBuilder.AddMolecule(appMoleculeBuilder);
+
          // add "ParticleApplication"-Tag required for Fraction Solid/Dissolved/Insoluble observer
```

Comment:
that didn't work due to our not optimal applications-BB structure (same formulation may appear multiple times)
(I was trying to create the Insoluble drug only once for the whole application, but with the current applications-BB structure it's quite hard. So the insoluble drug is still defined for every particle bin (which is actually useless and just consumes a compuatation time).
If we change our applications-BB structure in the future, we should get back again here and improve model structure.

Reply:
Do we have an issue for that in Github? I know there was one in YT but probably was not transferred

Reply:
I mean yes, but I could not find it on Github...